src/components/forms/lib/handleDemoSubmit.js

```diff
@@ -75,6 +75,13 @@ export const handleDemoSubmit = async ({
   const parsedFormValues = { isCompany, email, size, value };
 
   const response = await submitToHubspot(parsedFormValues);
+  if (response.status === 400) {
+    setTimeout(() => {
+      setFormStatus(INVALID_EMAIL);
+    }, 1000);
+    return;
```

Comment:
```suggestion
    const error = await response.json();    
    if (isInvalidEmailError(error)) {
      setFormStatus(INVALID_EMAIL);
      return;
    }
```

Not all errors are `INVALID_EMAIL` errors, so we need to check that as well.

Reply:
@morloy feedback addressed in pair 🍐 session with @marianacelyvelasquez 🍫